### PR TITLE
changed the font and color of amount placeholder and text

### PR DESCRIPTION
**Pull request by sandy-fyle**
image

image
-----------------------------------------------------------------------------------------------------------------------------------------
image

image

### CODE REVIEW

File: src/app/shared/components/fy-number/fy-number.component.scss [hunk 1]
```diff
@@ -3,15 +3,17 @@
 .fy-number {
   &--input {
     border: 0;
-    font-size: 14px;
-    font-weight: 400;
+    font-size: 20px;
+    font-weight: 600;
```
Comment (Dimple16) [inline src/app/shared/components/fy-number/fy-number.component.scss:7]: 500?

Comment (Dimple16) [reply src/app/shared/components/fy-number/fy-number.component.scss:7]: check what Aditi had mentioned in the doc

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/fy-number/fy-number.component.scss (hunk 1 above)
Comment (sandy-fyle) [reply src/app/shared/components/fy-number/fy-number.component.scss:7]: in figma design, 400 is mentioned but with font weight 400 it doesn't look the same as notion doc ss

Comment (Dimple16) [reply src/app/shared/components/fy-number/fy-number.component.scss:7]: 500 should do, as this is font-weight - `medium`

**Review by Dimple16 — CHANGES_REQUESTED** [review]

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**